tools/remote-file-navigator/README.md

```diff
@@ -0,0 +1,165 @@
+export PATH=$PATH:/usr/local/go/bin
+EOF
+source .bashrc
+apt install libx11-dev  libxrandr-dev \
+    libxcursor-dev \
+    libgl1-mesa-dev \
+    libxinerama-dev \
+    libxi-dev \
+    pkg-config \
+    xorg-dev -y
+# adjust to include a branch if required
+git clone https://github.com/gravitational/rev-tech 
+cd rev-tech/tools/remote-file-navigator/
+go mod init remote-file-nav
+go mod tidy
+#Adjust arch to amd64 if on that platform
+CGO_ENABLED=1 GOARCH=arm64 go build
+```
+
+## Running:
+
+```bash
+./remote-file-nav
+```
+
+
+## Quick Start
+
+### Connecting to a Remote Server
+
+1. **Using SSH Config (Recommended)**
+   - Ensure "Use SSH config" is checked
+   - Enter your SSH host alias in the **Host** field (e.g., `myserver` or `se-ag-dev-east.a4232.teleportdemo.com`)
+   - Click **Show Hosts** to see available hosts from your `~/.ssh/config`
+   - Enter **User** and **Pass** (password/passphrase) if required
+   - Click **Connect**
+
+2. **Manual Connection**
+   - Uncheck "Use SSH config"
+   - Enter the full hostname in **Host**
+   - Enter **User**, **Pass**, and optionally specify an SSH **Key** path
+   - Click **Connect**
+
+### Uploading Files (Local → Remote)
+
+1. Connect to a remote server
+2. Navigate to the desired **remote destination** directory using the right pane
+3. In the **left pane** (Local Files), check the boxes next to files/folders you want to upload
+4. Click **Upload ⬆** button (shows count of selected items)
+5. For directories, confirm the recursive upload when prompted
+
+### Downloading Files (Remote → Local)
+
+1. Connect to a remote server
+2. Navigate to the desired **local destination** directory using the left pane
+3. In the **right pane** (Remote Files), check the boxes next to files/folders you want to download
+4. Click **Download ⬇** button (shows count of selected items)
+5. For directories, confirm the recursive download when prompted
+
+### Navigation
+
+| Action | How To |
+|--------|--------|
+| Go up one directory | Click **Up** button or **← Up** |
+| Go to home directory | Click **Home** button or **🏠 Home** |
+| Enter a directory | Click on the folder name |
+| Go to specific path | Edit the **Path** field and press Enter or click **Go** |
+| Sort files | Click **Name**, **Size**, or **Date** column headers |
+| Show hidden files | Check **Show Hidden** |
+
+### Managing Files
+
+- **Delete local files**: Select files with checkboxes → Click **Delete**
+- **Delete remote files**: Select files with checkboxes → Click **Delete**
+- **Open terminal**: Click **Terminal** to launch an SSH session in your system terminal
+
+### Saving Connections
+
+1. Configure your connection settings (Host, User, Key, etc.)
+2. Click **Save** and enter a name for the connection
+3. Access saved connections from the dropdown menu
+4. Use **Load**, **Clear**, or **Delete** to manage saved settings
+
+> ⚠️ **Note**: Passwords are never saved for security reasons
+
+### Connecting to Teleport-Protected Servers
```

Comment:
ah i guess this answers the above Qs. I would shift this to the top since its meant to be the "happy path" usage from our standpont.